includes/class-activitypub.php

```diff
@@ -28,7 +28,7 @@ public static function init() {
 		\add_action( 'wp_trash_post', array( self::class, 'trash_post' ), 1 );
 		\add_action( 'untrash_post', array( self::class, 'untrash_post' ), 1 );
 
-		\add_action( 'init', array( self::class, 'add_rewrite_rules' ), 11 );
+		self::add_rewrite_rules();
```

Comment:
I would prefer to keep this as an action because I need to unhook it on dotcom due to our different rewrite requirements.

Does it cause problems elsewhere?

Reply:
done